Content.Server/GameObjects/Components/Mobs/Body/Limb.cs

```diff
@@ -0,0 +1,322 @@
+            reflectionManager = IoCManager.Resolve<IReflectionManager>();
+            CurrentHealth = MaxHealth;
+            Statuses = new List<LimbStatus>();
+            Owner = owner;
+            BodyOwner = body;
+            _seed = new Random(DateTime.Now.GetHashCode());
+            Organs = new List<Organ>();
+            foreach (var key in _organProt)
+            {
+                var prot = PrototypeManager.Index<OrganPrototype>(key);
+                var organ = prot.Create();
+                organ.Initialize(owner, body);
+                Organs.Add(organ);
+            }
+            if (reflectionManager.TryParseEnumReference(TargetKey, out var @enum))
+            {
+                AttackTarget = (AttackTargetDef)@enum;
+            }
+            foreach(var _addTargetKey in _addTargetKeys)
+            {
+                if (reflectionManager.TryParseEnumReference(_addTargetKey.Key, out var @anotherEnum))
+                {
+                    _addTargets.Add(new AdditionalTarget((AttackTargetDef)@anotherEnum, _addTargetKey.Value));
+                }
+            }
+        }
+
+        public LimbRender Render() //TODO
+        {
+            var color = new Color(128, 128, 128);
+            switch (State)
+                {
+                case LimbState.Healthy:
+                    color = new Color(0, 255, 0);
+                    break;
+                case LimbState.InjuredLightly:
+                    color = new Color(255, 255, 0);
+                    break;
+                case LimbState.Injured:
+                    color = new Color(255, 165, 0);
+                    break;
+                case LimbState.InjuredSeverely:
+                    color = new Color(255, 0, 0);
+                    break;
+                case LimbState.Missing:
+                    color = new Color(128, 128, 128);
+                    break;
+                }
+            return new LimbRender(TexturePath, color);
+
+        }
+
+        public void HandleGib()
+        {
+            if(directOrganDamage) //don't spawn both arm and hand, leg and foot
+            {
+                return;
+            }
+            if (_seed.Prob(0.7f))
+            {
+                foreach (var organ in Organs)
+                {
+                    organ.HandleGib();
+                }
+            } 
+            else 
+            {
+                SpawnPrototypeEntity();
+            }
+        }
+
+        public void HandleDecapitation(bool spawn)
+        {
+            State = LimbState.Missing;
+            foreach (var organ in Organs)
+            {
+                organ.State = OrganState.Dead;
+            }
+            foreach (var child in Children)
+            {
+                child.HandleDecapitation(childOrganDamage);
+            }
+            if (spawn)
+            {
+                SpawnPrototypeEntity();
+            }
+        }
+
+        private void Dispose()
```

Comment:
This isn't BYOND, you don't have to explicitly clear references. The GC can handle it.
Also you didn't implement `IDisposable` which you should if you're gonna have a `Dispose` method but that's kinda moot here.